scheduler/src/test/scala/com/sky/kms/e2e/SchedulerIntSpec.scala

```diff
@@ -48,7 +49,10 @@ class SchedulerIntSpec extends SchedulerIntSpecBase {
           cr.key should contain theSameElementsInOrderAs schedule.key
           cr.value should contain theSameElementsInOrderAs schedule.value.get
           cr.timestamp shouldBe time.toInstant.toEpochMilli +- Tolerance.toMillis
-          cr.headers().toArray.map(h => h.key() -> h.value()).toMap should contain theSameElementsAs schedule.headers
+          cr.headers().toArray.map(h => h.key() -> h.value().toList) should contain theSameElementsAs
+            schedule.headers.map {
+              case (k, v) => (k, v.toList)
```

Comment:
Due to the values being `Array[Byte]`, the equality test would always fail, even if they were equivalent (due to array equality basically being rubbish in scala).